lib/assessment.js

```diff
@@ -8,25 +8,80 @@
 // Question 1
 // Write a callback for reduce that selects
 // the minimum value in the array
-var min = function min() {
 
+var min = function min(previousValue, currentValue, index, array) {
+  return previousValue < currentValue ? previousValue : currentValue;
 };
 
+console.log(reduce(numbers, min));
+console.log(numbers.reduce(function (previousValue, currentValue) {
+  return previousValue < currentValue ? previousValue : currentValue;
+}));
+
+
+};
+
+// I used Antony's Max Example of code and switched the operators.
+// I do not think I could logic out the whole thing as concisely as this.
+// I cannot write this code from scratch myself but I can read this and edit it
```

Comment:
We don't need your code to be as concise as Antony's. He has been doing this for decades!